common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -7,6 +7,23 @@ import scala.util.Try
 
 import com.gu.mediaservice.model.{FileMetadata, ImageMetadata}
 
+object SupplierCategory extends Enumeration {
+  type SupplierCategory = Value
+  val Sport = Value("Sport")
+  val Entertainment = Value("Entertainment")
+  val Unknown = Value("Unknown")
+
+  // These category codes are now deprecated but still populated
+  // See: https://www.iptc.org/std/photometadata/documentation/GenericGuidelines/index.htm#!Documents/guidelineformappingcategorycodestosubjectnewscodes.htm
+  def create(category: String) = category match {
```

Comment:
The full list is probably at that URL. 

I suspect this might be useless for anyone else except sport folks *and* is a deprecated standard. They also seem to be used as a way for folks to handle the stream of incoming images (rather than looking very far back), so we may not need to reindex old images if we're lucky.